snippet_uiautomator/uiobject2.py

```diff
@@ -165,38 +165,25 @@ def to(
       )
 
 
-class _Gesture:
+class _Swipe:
   """Performs a gesture in a specific direction on a specific UiObject2."""
 
   def __init__(
       self,
       ui: snippet_client_v2.SnippetClientV2,
       selector: byselector.BySelector,
-      action: str,
   ) -> None:
     self._ui = ui
     self._device = self._ui._device  # pylint: disable=protected-access
     self._selector = selector
-    self._action = action
 
   def _perform_gesture(
```

Comment:
Rename to `_perform_swipe`?

Reply:
Sure, commit added.

Reply:
Done
<!-- Reviewable comment -ODkZHBR6tLVivA38OP0:bb6un1j -->
<!-- Sent from Reviewable.io -->